src/nasuta-sebastian/Fsd.Sebastian.Cs/Fsd.Sebastian.Cs2/AnimalFactory.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Fsd.Sebastian.Cs2.Animals;
+using System;
+using System.Collections.Generic;
+
+namespace Fsd.Sebastian.Cs2
+{
+    public class AnimalFactory
+    {
+        private List<Animal> _animals;
+
+        private int numberOfAnimals;
+
+        private string animalType;
+
+        public void Start()
+        {
+            _animals = new List<Animal>();
+
+            animalType = "cat";
+            numberOfAnimals = SetAnimalsCount(animalType);
+            for (int i = 0; i < numberOfAnimals; i++)
+                _animals.Add(new Cat(ConsolePresenter.GetString($"{animalType} name: ")));
+
+            animalType = "dog";
+            numberOfAnimals = SetAnimalsCount(animalType);
+            for (int i = 0; i < numberOfAnimals; i++)
```

Comment:
pomiędzy przypisaniem i for powinna być pusta linia (przechodzisz do nowego bloku kodu)